src/main/java/frc/robot/Robot.java

```diff
@@ -415,6 +415,7 @@ public void teleopInit() {
     Climb.setBrakeMode();
     Intake.setTiltBrakeMode();
     Intake.setLimit();
+    Shooter.SetPID(0.2, 0.0005, 3.5, 0.05, 200.0);
```

Comment:
Split out constants into some file. RobotMap? SCREAMING_SNAKE_CASE for constants.